svf-llvm/lib/LLVMUtil.cpp

```diff
@@ -1040,15 +1038,17 @@ const std::string SVFValue::toString() const
     {
         rawstr << "BasicBlock: " << bb->getName() << " ";
     }
+    else if (LLVMModuleSet::getLLVMModuleSet()->getLLVMValue(this) == nullptr)
+    {
+        assert((SVFUtil::isa<SVFInstruction>(this) ||
+                SVFUtil::isa<SVFBasicBlock>(this)) && "this is not a manually created call inst by ExtAPI!");
```

Comment:
This `SVFUtil::isa<SVFBasicBlock>(this))` will never be true, because it went into line 1037 before reaching here.